force-app/volunteers/reports/VolunteerReports/RecentVolunteers.report-meta.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Report xmlns="http://soap.sforce.com/2006/04/metadata">
+    <columns>
+        <field>FIRST_NAME</field>
+    </columns>
+    <columns>
+        <field>LAST_NAME</field>
+    </columns>
+    <columns>
+        <field>EMAIL</field>
+    </columns>
+    <columns>
+        <field>PHONE1</field>
+    </columns>
+    <columns>
+        <field>Contact.LastVolunteerDate__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>Contact.VolunteerHours__c</field>
+    </columns>
+    <description>Contacts whose Last Volunteer Date is within the last 120 days.</description>
```

Comment:
I see the open issue about Start Date on Volunteer Hours Logs and how that's affecting this report. Thanks for flagging that. I'll look into it.